src/java/org/apache/cassandra/cache/UnweightedCacheSize.java

```diff
@@ -0,0 +1,42 @@
+/*
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ *
+ */
+package org.apache.cassandra.cache;
+
+/**
+ * Represents an unweighted cache size.
+ */
+public interface UnweightedCacheSize
+{
+    /**
+     * Returns the maximum number of entries in this cache.
+     */
+    int maxEntries();
```

Comment:
Should this be renamed to `getMaxEntries`?

Reply:
The naming of these methods basically copies these in `CacheSize`. I would keep it like it is. In CacheSize there is `capacity` / `getCapacity` etc ...